src/platform/renesas/RenesasConfig.cpp

```diff
@@ -46,14 +46,16 @@ CHIP_ERROR RenesasConfig::ReadConfigValueStr(Key key, char * buf, size_t bufSize
     return ReadConfigValueBin(key, buf, bufSize, outLen);
 }
 
-CHIP_ERROR RenesasConfig::ReadConfigValueBin(Key key, uint8_t * buf, size_t bufSize, size_t & outLen)
-{
-    return ReadConfigValueBin(key, static_cast<void *>(buf), bufSize, outLen);
-}
-
 CHIP_ERROR RenesasConfig::ReadConfigValueBin(Key key, void * buf, size_t bufSize, size_t & outLen)
 {
-    return CHIP_DEVICE_ERROR_CONFIG_NOT_FOUND;
+    char key_str[MTB_KVSTORE_MAX_KEY_SIZE] = { 0 };
+    snprintf(key_str, sizeof(key_str), "%08x", static_cast<unsigned long>(key));
```

Comment:
As sicussed, please keep these as unimplemented and implement `ConfigurationManagerImpl::GetRebootCount` and `StoreRebootCount` instead (`ConfigurationManagerImpl::Init` will need fixed, because this key will never exist)

Reply:
updated